### DISCUSSION TITLE
Auth  keep getting timed out.

### DISCUSSION THREAD

**Discussion by TheRed-Jester** [ThioJoe/YT-Spammer-Purge#645] [Q&A] [answered]
I was using SpammerPurge for total 5 days out of which it worked fine for 4 days. But from today I keep getting 'Connection to oauth2.googleapis.com timed out.' . I have tried deleting the program and downloading different version 2-3 times. Redownloaded Json file. But the problem is still there. What can I do so that it works fine again. Thanks in advance

**Comment by Kurdakov** [comment, marked as answer]
the same problem is reported on issue page, some problems on google side

**Reply by TechStudent10** [reply]
Dup of #643

**Comment by TechStudent10** [comment]
Duplicate of https://github.com/ThioJoe/YT-Spammer-Purge/issues/643

**Reply by TechStudent10** [reply]
Oh wait I just realized you created your account an hour ago sooo...

Welcome to GitHub

**Reply by TechStudent10** [reply]
like seriously this is what happens on GitHub

**Reply by TheRed-Jester** [reply]
LOL, Well it started working fine. Thaks mate.
